docs/conf.py

```diff
@@ -213,7 +226,45 @@
 # Use a code highlighting style that meets the WCAG AA contrast standard
 pygments_style = "default"
 
+if os.environ.get("READTHEDOCS"):
+    # Building on Read the Docs
+    hoverxref_api_host = "https://readthedocs.org"
+
+    if os.environ.get("PROXIED_API_ENDPOINT"):
+        # Use the proxied API endpoint
+        hoverxref_api_host = "/_"
```

Comment:
[optional] I'm wondering if it would be worth updating the comment to describe what a proxied API endpoint is, since this might be unfamiliar teminology for many of us (including me), and the sphinx-hoverxref source code and docs doesn't really describe what this is doing.

Reply:
I honestly don't understand it myself, I just copied it over from `sphinx-hoverxref`'s `conf.py`.

https://github.com/readthedocs/sphinx-hoverxref/blob/003b84fee48262f1a969c8143e63c177bd98aa26/docs/conf.py#L81-L95

Reply:
This is a RTD thing to avoid a CSRF block. Without this, the request is done to readthedocs.org. When using "_/" the request is done to the same domain where the documentation is hosted.

Under RTD "_/" has a special meaning. It means to hit the Django app instead of the "hosting app".

Not sure if that clarifies. I'm on the phone, but I can expand later if needed.